.ci/packaging/request_to_obs.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
```

Comment:
Can we have a little bit of explanation of what the script does here at the top of the file pls? thx.